docs/src/developers_guide/contributing_documentation_easy.rst

```diff
@@ -0,0 +1,95 @@
+the improvement yourself)!
+
+First time only steps
+^^^^^^^^^^^^^^^^^^^^^
+ 
+1. Create a `GitHub <https://github.com/>`_ account.
+
+2. Complete the Scitools Contributor License Agreement (`link to Google Form
+   <https://docs.google.com/forms/d/e/1FAIpQLSfd0tdE-DcJOXh8ej_7T93IizwJFYBFyRWYQOi2A8QRaKwykA/viewform>`_).
+   This is a one-off requirement for anyone who wishes to contribute to a
+   Scitools repository - including the documentation.
+
+Steps to complete each time you propose changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Navigate to documentation page that you want to edit (on this site).
+
+2. Click the 'Edit on GitHub' button at the top right of the page.
+
+.. image:: edit_on_github.png
+
+3. In the resulting GitHub page select **main** from the drop-down menu near
+   the top left of the page (to the left of the 'iris / docs / src / …' links)
+   if it isn't already. This changes the branch to main.
+
+.. image:: find_main.png
+
+4. Click the pencil symbol near the top right (to the right of the 'Raw' and
+   'Blame' buttons).
+
+.. image:: edit_button.png
+
+5. Make your edits! Try to strike a balance between informing the audience
+   enough that they understand and overwhelming them with information.
+
+.. note::
+
+    You may see the following message at the top of the edit page, informing you
+    that GitHub has created you your own 'fork' (or copy) of the project as a
+    precursor to allowing you to edit the page. Your changes will be merged into
+    the main version of the documentation later.
+
+    .. image:: fork_banner.png
+
+6. Scroll to the bottom of the edit page and enter some appropriate information
+   in the two boxes under 'Propose changes'. You can just keep the default text
+   if you like or enter something more specific - a short sentence explaining
+   what's changed is fine. Then click the 'Propose changes' button.
+
+.. image:: propose_changes.png
+
+7. In the resulting page titled 'Pull Request', write a brief description of
+   what you've changed underneath the following three lines:
+
+.. code::
+
+    ### Description
+    <!-- Provide a clear description about your awesome pull request -->
+    <!-- Tell us all about your new feature, improvement, or bug fix -->
+
+Describing what you've changed and why will help the person who reviews your changes.
+
+.. image:: pull_request.png
+
+8.	Click the 'Create pull request' button.
+
+What happens next?
+^^^^^^^^^^^^^^^^^^
+
+Someone else who works on Iris will review your changes (this happens for
+everyone who makes changes to Iris or its documentation). The reviewer might
+make comments or ask questions (don't worry about missing these, GitHub will
+email you to let you know). You can respond to these comments underneath where
+they appear in GitHub.
+
+Once you've worked everything out together, the reviewer will merge your changes
+into the main version of the documentation so that they're accessible for
```

Comment:
\`\`main``

Reply:
I was intentionally leaving this as a word they know they understand, to link to thinking about the "main branch" later - can see the argument for using the language explicitly in a technical way though. What do you think?

Reply:
No worries, sounds good to me :+1:

Reply:
How about a compromise...
`Once you've worked everything out, the reviewer will merge your documentation changes so that they're published and accessible for everyone to benefit from`

Reply:
I think
`...the reviewer will merge your documentation changes so that ...`
relies on some understanding of our special meaning of `merge` that I don't want to rely on, whereas
`...the reviewer will merge your changes into the main version...`
uses the everyday meaning of merge.

So I'd go for what's currently there, or
`...will publish your changes so that they're accessible...`

I'm personally a fan of tying the "main version" to where they've seen "main" in the context of branching earlier, but not enough to fight for it.